services/shhext/chat/encryption_test.go

```diff
@@ -866,3 +876,88 @@ func (s *EncryptionServiceTestSuite) TestRefreshedBundle() {
 	s.Equal(bobBundle2.GetSignedPreKeys()[bobInstallationID].GetSignedPreKey(), x3dhHeader2.GetId())
 
 }
+
+func (s *EncryptionServiceTestSuite) TestMessageConfirmation() {
```

Comment:
Do you think we could write this test as a table driven test?

Reply:
I can try that, I will probably do in a separate PR if it's ok